frontend/src/components/print/PrintConfigurator.vue

```diff
@@ -208,6 +216,13 @@ export default {
 
       return contents
     },
+    isMultiple(content) {
+      const contentComponent = this.contentComponents[content.type]
+
+      const multipleFlagSet = contentComponent.design.multiple
+      const hasMultiplePeriods = content.options?.periods?.length > 1
+      return multipleFlagSet || hasMultiplePeriods
+    },
```

Comment:
This is not used, right?

Reply:
yes.
So you like this variant more?
(see https://github.com/ecamp/ecamp3/pull/3516#discussion_r1262805438)

Reply:
done